### ISSUE TITLE
Suggestion: Area Labels for Spirit Tree Map

### ISSUE THREAD

**Issue by Jin-Jiyunsun** [MJHylkema/spirit-tree-map#9]
Here is a rough mock up of my suggestion, that I threw together real quick, basically labels on the main areas? as a toggle probably?

![image](https://github.com/MJHylkema/spirit-tree-map/assets/37377657/226419d4-2cb1-4cc2-a060-f19e6ff0e094)

**Comment by MJHylkema** [comment]
Would you prefer labels per area or per teleport? I have already started working on a version that has toggle-able labels per tree. These what basically display the name you see when you hover a tree, but at all times on the map. 
Would that satisfy your request?

**Comment by Jin-Jiyunsun** [comment]
> Would you prefer labels per area or per teleport? I have already started working on a version that has toggle-able labels per tree. These what basically display the name you see when you hover a tree, but at all times on the map. Would that satisfy your request?

Yup thats even better, I was going to suggest it as well but I figured it might get a bit cramped with all the names.

**Comment by Jin-Jiyunsun** [comment]
Any update on progress on this?

**Comment by MJHylkema** [comment]
Sorry, haven't had a lot of free time lately. Back from holiday now, so I will get onto this pronto. 👍🏼
